samples/snippets/feature_store_service/create_feature_sample_test.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from uuid import uuid4
+
+import create_feature_sample
+import pytest
+
+import helpers
+
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+
+
+@pytest.fixture(scope="function", autouse=True)
+def teardown(teardown_feature):
+    yield
+
+
+def test_ucaip_generated_create_feature_sample_vision(capsys, shared_state):
+    featurestore_id = "sample_featurestore"
+    entity_type_id = "sample_entity_type"
```

Comment:
same comment about perm resources

Reply:
Added perm_ prefix to them, now we have `perm_sample_featurestore` and `perm_sample_entity_type`